misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindgenCommand.java

```diff
@@ -115,13 +117,13 @@ public void execute() {
         }
 
         if (classNames == null) {
-            setOutError("One or more class names should be specified to generate the Ballerina bindings.");
+            setOutError("error: one or more class names are required");
```

Comment:
Can we move "error: " prefix to the `setOutError` function ?

Reply:
Done.